### ISSUE TITLE
Integrate FocusTrap to ModalButtons

### ISSUE THREAD

**Issue by ptbrowne** [cozy/cozy-ui#1481]
In cozy-store, we use FocusTrap inside modals to be sure
that tabbing while a modal is shown will focus around
the modal content.

It could be interesting to have this automatically
integrated to cozy-ui directly, for example having
it automatically around `<ModalButtons />`.

**Comment by y-lohse** [comment]
>  Should focus trap be put at modal level or only on the action buttons ?

One thing I noticed in #1482 is that you can't focus the close button with the keyboard anymore, so I think using it a the modal level would make more sense imo.

**Comment by ptbrowne** [comment]
Yes, I was not very concerned since you can use the Esc key to close the modal. To me, cycling only through the action buttons is more natural than on the action buttons + close buttons. I have no strong feeling about this though.
